src/process/slice/index.js

```diff
@@ -0,0 +1,5 @@
+import userReducer from 'process/redux/userSlice';
```

Comment:
Also move the actual slices from redux folder to slices folder...

Reply:
okay